### ISSUE TITLE
Public methods subscribers

### ISSUE THREAD

**Issue by wideawake** [greenrobot/EventBus#384]
Hi,

I have some subscribers with protected visibility. While in development everything works fine, with the signed apk these subscribers are not found from EventBus thus events posted are not delivered.

Adding the apt plugin to create the index (which in my case did not generate the class specified in the config) and performing Make project, I get errors for this modifiers. Using the annotation processor no warnings are fired.

Is there any method to avoid in the future this kind of sad situation that I fell in?

Thank you

**Comment by greenrobot-team** [comment]
Yes, fix the error output. Error messages do not display with annotationProcessor API of Android Gradle Plugin. -ut
